README.md

```diff
@@ -113,12 +113,12 @@ struct Item {
 }
 
 fn main() -> Result<(), db_type::Error> {
-    let mut builder = DatabaseBuilder::new();
+    let mut models = Models::new();
```

Comment:
I'd add a comment suggesting to do it in lazy_static or smth

Reply:
Very good suggestion, thank you! Done.